test/nodetool/utils.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import pytest
+import subprocess
+
+
+def check_nodetool_fails_with(
+        nodetool,
+        nodetool_args: tuple,
+        nodetool_kwargs: dict,
+        cassandra_error: str,
+        scylla_error: str):
+
+    with pytest.raises(subprocess.CalledProcessError) as e:
+        nodetool(*nodetool_args, **nodetool_kwargs)
+
+    err_lines = e.value.stderr.rstrip().split('\n')
```

Comment:
I'm confused, isn't e scoped to the with statement?

Reply:
Python scopes are funky. In fact, if I move this code into the scope, I get an error that e cannot be used in said scope. I think this is beacause the enclosed code is expected to throw and the exception is only guaranteed to be ready only after the scope exited.